src/App.jsx

```diff
@@ -44,49 +46,70 @@ function PrivateRoute({ children, ...rest }) {
   );
 }
 
+// grab from domain or some service
+const ORGANIZATION_ID = "1";
+
 function App() {
+  const [org, setOrg] = useState();
+  useEffect(() => {
+    Organization.init(ORGANIZATION_ID)
+      .then((org) => setOrg(org))
+      .catch((error) => {
+        console.error(error);
```

Comment:
should we do a snackbar error here ?

Reply:
Hmm there's not currently a good way to set the snackbar while outside of the snackbar provider. Something to look at later

Reply:
Depends on how the app should respond to the app unable to run the init org method right?

If we’re dependent on these org hook being set in the provider, maybe we need a more serious error lol. 

Will the app work if the org is not set?

Reply:
Everything basically depends on the org being set in order to make certain calls. So right now I just have it redirecting to the organizer signup page if it doesn't exist